docs/index.asciidoc

```diff
@@ -225,20 +226,18 @@ The ingest document's `tags`.
 When the ingest document has a value for `tags` that cannot be coerced, it will be available in the event's `_tags` field.
 |=======================================================================
 
-Additionally, the following Elasticsearch IngestDocument Metadata fields are made available on the resulting event _if-and-only-if_ they were set during pipeline execution:
-
-:mcc-prefix: [@metadata][_ingest_document]
+Additionally, these {es} IngestDocument Metadata fields are made available on the resulting event _if-and-only-if_ they were set during pipeline execution:
 
 [cols="<1,<5",options="header"]
 |=======================================================================
 | {es} document metadata | {ls} field
 
-| `_id`                | `{mcc-prefix}[id]`
-| `_index`             | `{mcc-prefix}[index]`
-| `_routing`           | `{mcc-prefix}[routing]`
-| `_version`           | `{mcc-prefix}[version]`
-| `_version_type`      | `{mcc-prefix}[version_type]`
-| `_ingest.timestamp`  | `{mcc-prefix}[timestamp]`
+| `_id`                | `[@metadata][_ingest_document][id]`
+| `_index`             | `[@metadata][_ingest_document][index]`
+| `_routing`           | `[@metadata][_ingest_document][routing]`
+| `_version`           | `[@metadata][_ingest_document][version]`
+| `_version_type`      | `[@metadata][_ingest_document][version_type]`
+| `_ingest.timestamp`  | `[@metadata][_ingest_document][timestamp]`
```

Comment:
Fixes the problem where attributes were rendering literally (because of backticks), and makes the field info more readable from the GitHub source file as recommended in doc guidelines.